### DISCUSSION TITLE
Using multiple schemas does not work as expected

### DISCUSSION THREAD

**Discussion by ivari** [0no-co/gql.tada#178] [Q&A] [answered]
I have a feeling that I may be doing something wrong, so I thought I'd ask here.

My `tsconfig.json`
```
"plugins": [
  {
    "name": "@0no-co/graphqlsp",
    "schema": "admin_schema.graphql",
    "tadaOutputLocation": "src/graphql-admin-env.d.ts"
  },
  {
    "name": "@0no-co/graphqlsp",
    "schema": "core_schema.graphql",
    "tadaOutputLocation": "src/graphql-core-env.d.ts"
  }
]
```

My component
```
import { initGraphQLTada } from 'gql.tada'
import type { introspection } from 'src/graphql-core-env.d'

const graphql = initGraphQLTada<{
  introspection: introspection,
}>()

export const Tada = () => {
  const query = graphql(`
    query getSth {
      fields { id }
    }
  `)

  return null
}
```

This gives me an error `Cannot query field "fields" on type "QueryRoot".`, although query `fields` exists on core.
If I were to query `companies`, which exists on admin, but not on core, I have no error.

Interestingly enough, autocomplete uses admin, not core.

This behaviour does not change if I were to import admin instead of core.
This behaviour does not change if I import `graphql` straight from `gql.tada`.

**Comment by ivari** [comment]
Ah, it may have been more a question for the LSP plugin. I see a related discussion at https://github.com/0no-co/GraphQLSP/discussions/223

Then again, https://gql-tada.0no.co/get-started/installation#initializing-gql-tada-manually mentions multiple schemas.

**Comment by kitten** [comment, marked as answer]
Multi-Schema Support has been released in `gql.tada` 1.6.0 (+ `@0no-co/graphqlsp@^0.12.0`).
There's more information in the corresponding announcement devlog post: https://gql-tada.0no.co/devlog/2024-04-26

**Reply by HiggsWRX** [reply]
Wow this was fast. Thanks @kitten, much appreciated!
